server/ai_http.go

```diff
@@ -371,6 +381,29 @@ func handleAIRequest(ctx context.Context, w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	//backwards compatibility to old gateway api
```

Comment:
Do we actually need it? 

If yes, could you add a longer comment to describe more about it?

Reply:
This is now line 427. Expanded comment.  The main update made here is the gateway also adds the `protoVerAIWorker` to the request headers sent to the Orchestrator.  We are iterating on communication protocols with the AI jobs so I thought it would be good to establish a compatibility check and adjust the response as needed for a short while.  

This can be removed after a short update window (1-2 week?) after this PR is merged if would prefer to not have a backwards compatibility check in Gateway > Orchestrator requests.